css/index.css

```diff
@@ -0,0 +1,438 @@
+@import url('https://fonts.googleapis.com/css2?family=Gowun+Batang:wght@400;700&display=swap');
+
+body {
+  margin: 0;
+  border-width: 0;
+  box-sizing: border-box;
+  height: 100vh;
+
+  font-family: "Gowun Batang", serif;
+  font-weight: 400;
+  font-style: normal;
+
+  font-size: 16px;
+
+  line-height: 1.5;
+}
+
+h1 {
+  font-family: "Gowun Batang", serif;
+  font-weight: 700; 
+  line-height: 1; 
+  font-size: 36px;
+}
+
+h2 {
+  font-family: "Gowun Batang", serif;
+  font-weight: 700; 
+  line-height: 1.2; 
+  font-size: 24px;
+}
+
+.highlight {
+  color: #257650; 
+  font-weight: 700; 
+}
+
+.highlight-2{
+  color: #A4C2F7;
+  font-weight: 700; 
+  font-size: 36px;
+}
+
+.highlight-3{
+  color: #A4C2F7;
+  font-weight: 700; 
+  font-size: 24px;
+}
```

Comment:
**praise:** I like that you used classes for highlighting certain passages, instead of inline style attributes. This makes it easier to alter what a highlight looks like later, if desired.